docs/en/siem/index-api-overview.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[index-api-overview]]
+=== Index endpoint
+
+You use the index endpoint to create, get, and delete a signal index in a 
+{kib} space.
+
+You can only create a signal index when the user role has `manage` privileges 
+for both the {ref}/security-privileges.html[{es} cluster and the signals index].
```

Comment:
Is the "signals index" the name of SIEM or backend element? If so, keep the sentence as is, but if there's some noun confusion there. In its current usage, the word "signals" is genitive, implying that the signals posses the index. Switching to "signal index" would help clear up some of the confusion.